examples/tutorials/03_vision/02_face_follower.py

```diff
@@ -21,13 +21,12 @@
 '''
 
 import asyncio
-import sys
 import time
 
 import cozmo
 
 
-def follow_faces(robot):
+def follow_faces(robot: cozmo.robot.Robot):
```

Comment:
Looks like this script used to support having a KeyboardInterrupt. Does it still? If yes, we should have this line:

print("Press CTRL-C to quit")